topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,455 @@
+
+![schema of the whole process of training model and test.](../../images/galaxy-ludwig/explain_model_schema.png "Overview of the steps process to obtain the handwritten classification model and testing it.")
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+> <comment-title>Background</comment-title>
+>
+> [MNIST dataset](https://yann.lecun.com/exdb/mnist/) provides the database of handwritten digits
+> the database has a training set of 60,000 examples, and a test set of 10,000 examples.
+> at our planet and its environment for the ultimate benefit of all European citizens.
+> It is a subset of a larger set available from NIST.
+> The digits have been size normalized and centered in a fixed-size image.
+{:  .comment}
+
+# Files Format
+Before starting our hands-on, here is a brief explanation of the three files generated for the Ludwig Experiment tool.
+
+## Image_Files.zip 
+
+Since our model will learn directly from the PNG files containing the handwritten numbers, we need to upload these files as well. The images are stored in two main folders: one for training and one for testing. Another folder, named after the figure label, is used to store the figures (Figure 2).
+
+Notice that we are going to work with compressed file (.zip), Galaxy-Ludwig knows how to decompress the files and have them ready.
+
+![folder tree related to the hand-written digits picture](../../images/galaxy-ludwig/explain_images_file.png "File tree containing the images files used to training and testing the model")
+
+## MNIST_dataset.csv
+
+The MNIST dataset consists of images and their corresponding labels. For the purpose of this tutorial, mnist_dataset.csv file is created and contains three columns: image_path, label and, split.
+
+Briefly, the image_path column provides the file paths to the images that will be fed into the deep learning algorithm. The label column contains the correct classifications, ranging from 0 to 9, for the handwritten digits in the images. The split column indicates whether the data should be used for training (0) or testing (2) the model.
+
+![Dataset.csv file format snapshot](../../images/galaxy-ludwig/explain_dataset_format.png "Dataset.csv file format snapshot. features in order: file_path, label, split."){: width="50%" height="10%"}
+ 
+## Config.yaml
+
+The config.yaml file is crucial as it defines the entire structure of your machine learning experiment. This configuration file tells Ludwig how to process your data, what model to use, how to train it, and what outputs to generate.
+
+The rationale on how this file was constructed for this dataset is the following:
+i) The model takes images as input and uses a stacked convolutional neural network (CNN) to extract features.
+ii) It consists of two convolutional layers followed by a fully connected layer, with dropout applied to both the second convolutional layer and the fully connected layer to reduce overfitting. 
```

Comment:
Do we have other layers such as `maxpooling` and `normalisation` used in the architecture of stacked CNN? If yes, please mention them.